dojo/tools/wpscan/parser.py

```diff
@@ -70,46 +71,65 @@ def get_findings(self, file, test):
         tree = json.load(file)
 
         report_date = None
-        if 'start_time' in tree:
-            report_date = datetime.utcfromtimestamp(tree.get('start_time'))
+        if "start_time" in tree:
+            report_date = datetime.utcfromtimestamp(tree.get("start_time"))
 
         dupes = dict()
         # manage plugin findings
-        for plugin in tree.get('plugins', []):
-            node = tree['plugins'][plugin]
-            self.get_vulnerabilities(report_date, node.get('vulnerabilities'), dupes, node, plugin)
+        for plugin in tree.get("plugins", []):
+            node = tree["plugins"][plugin]
+            self.get_vulnerabilities(
+                report_date,
+                node.get("vulnerabilities"),
+                dupes,
+                node,
+                plugin,
+                detection_confidence=tree["plugins"][plugin].get("confidence"),
```

Comment:
```suggestion
                detection_confidence=node.get("confidence"),
```

This should work as well but would be easier to read.